search-sentiments-map/public/scripts/trends-script.js

```diff
@@ -57,21 +54,31 @@ function setTopTrends() {
 }
 
 /**
- * Shows the remaining 10 trending topics if the user clicks on 'See More' and
- * hides the last 10 if the user clicks on 'See Less.'
+ * Shows the trending topics over the 7th if the user clicks on 'See More' and
+ * hides those if the user clicks on 'See Less.'
  */
 function showMoreOrLess() {
   const showMoreOrLessToggleItem = document.getElementById(SHOW_MORE_OR_LESS_ID);
   const trendElements = document.querySelectorAll('#trends-list li');
   if (showMoreOrLessToggleItem.innerText === TOGGLE_SHOW_MORE) {
-    for (var i = 10; i < trendElements.length - 1; i++){
+    for (let i = NUM_SHOWN; i < trendElements.length - 1; i++){
       trendElements[i].className  = CLASSNAME_SHOWN;
     }
     showMoreOrLessToggleItem.innerText = TOGGLE_SHOW_LESS;
   } else {
-    for (var i = 10; i < trendElements.length - 1; i++) {
+    for (let i = NUM_SHOWN; i < trendElements.length - 1; i++) {
       trendElements[i].className  = CLASSNAME_HIDDEN;
     }
     showMoreOrLessToggleItem.innerText = TOGGLE_SHOW_MORE;
   }
 }
+
+/** 
+ * Update title when a trending topic is selected.
```

Comment:
A little more descriptive in the comment? You want to say "title of a topic" or some such. Title could also refer to the title of an article or title.

Reply:
ok